myriad-scheduler/src/test/java/org/apache/myriad/MyriadTestModule.java

```diff
@@ -78,14 +78,7 @@ protected void configure() {
   @Provides
   @Singleton
   ExecutorCommandLineGenerator providesCLIGenerator(MyriadConfiguration cfg) {
-    ExecutorCommandLineGenerator cliGenerator = null;
-    MyriadExecutorConfiguration myriadExecutorConfiguration = cfg.getMyriadExecutorConfiguration();
-    if (myriadExecutorConfiguration.getNodeManagerUri().isPresent()) {
```

Comment:
One doesn't necessarily need a NodeManagerUri, it's optional - Docker or static install are valid options.